.github/workflows/tests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - macos-latest
+          # - macos-latest # TODO: fix
```

Comment:
Is this still TODO? If so, why?

Reply:
I think at some point `setup-ocaml` had some problems with `depext` due to opam 2.1, but I'll revert it now. Based on other CI runs now, it should be fixed.